packages/transport-bridge/src/core.ts

```diff
@@ -120,11 +120,12 @@ export const createCore = (apiArg: 'usb' | 'udp' | AbstractApi, logger?: Log) =>
         acquireInput: Omit<AcquireInput, 'previous'> & {
             previous: Session | 'null';
             signal: AbortSignal;
-        },
+        } & { sessionOwner: string },
     ) => {
         const acquireIntentResult = await sessionsClient.acquireIntent({
             path: acquireInput.path,
             previous: acquireInput.previous === 'null' ? null : acquireInput.previous,
+            sessionOwner: acquireInput.sessionOwner,
```

Comment:
NOTE: is it necessary to have `sessionOwner` in both `acquireIntent` and `acquireDone`?

Reply:
nope, not needed. was needed before your refactoring, not anymore